app/validation/validators.py

```diff
@@ -19,8 +19,8 @@ def __init__(self, message=None):
 
     def __call__(self, form, field):
         try:
-            Decimal(field.raw_data[0])
-        except (ValueError, TypeError, InvalidOperation):
+            Decimal(field.raw_data[0].replace(',', ''))
```

Comment:
We should use Babel to return the thousands separators for the locale (get_group_symbol()). In the DecimalPlaces validator we should also use babel's get_exponential_symbol to get the decimal place symbol for the locale. We should make DEFAULT_LOCALE a global variable that can be used by jinja_filters.py, cusotm_fields.py and validators.py.